apps/discord-bot/.gitignore

```diff
@@ -0,0 +1 @@
+config/
```

Comment:
should we still need to keep the *.example files?

Reply:
yes, the *.example is used as template for the build config

Reply:
then we need to update the ignore rule? otherwise, all files in `/config` will be removed from git